### DISCUSSION TITLE
Adding JS & TS

### DISCUSSION THREAD

**Discussion by DavidTxx** [div-bargali/Data-Structures-and-Algorithms#911] [Ideas]
Would be nice to add JS & TS languages. 
Available there
https://github.com/sularome/TypeScript-Algorithms-and-Data-Structures
and there:
https://github.com/PacktPublishing/Learning-JavaScript-Data-Structures-and-Algorithms-Third-Edition/tree/master/LearningJavaScriptDataStructuresandAlgorithmsThirdEdition_Code

**Comment by div-bargali** [comment]
> Would be nice to add JS & TS languages. Available there https://github.com/sularome/TypeScript-Algorithms-and-Data-Structures and there: https://github.com/PacktPublishing/Learning-JavaScript-Data-Structures-and-Algorithms-Third-Edition/tree/master/LearningJavaScriptDataStructuresandAlgorithmsThirdEdition_Code

@DavidTxx We would accept new languages as well, in case you are interested, you can contribute.
